terraform/variables.tf

```diff
@@ -141,6 +141,12 @@ variable "acm_secret_type" {
   type        = string
 }
 
+variable "acm_source_repository_fqdns" {
+  description = "FQDN of source repository for Config Sync to allow in the Network FIrewall Policy"
+  type        = list(string)
+  default     = ["github.com", "gitlab.com"]
```

Comment:
Considering that we require other ACM variables, we can avoid adding a default here.